src/main/scala/com/atomist/rug/spi/Handlers.scala

```diff
@@ -42,41 +49,60 @@ object Handlers {
         case _ => throw new IllegalArgumentException(s"Cannot derive Instruction from '$name'.")
       }
     }
+
     case class Generate(detail: Detail) extends Instruction
+
     case class Edit(detail: Detail) extends Instruction
+
     case class Review(detail: Detail) extends Instruction
+
     case class Execute(detail: Detail) extends Instruction
+
     case class Command(detail: Detail) extends Instruction
+
     case class Respond(detail: Detail) extends Instruction with Callback
+
   }
 
   case class MavenCoordinate(group: String,
-                             artifact: String,
-                             version: Option[String])
+                             artifact: String) {
```

Comment:
Indeed. Who needs versions, eh? 

It's not (currently) allowed for users specify the version of a Rug to run from their Plan. Versions come from the manifest.yml.